src/server/game/Entities/Creature/Creature.cpp

```diff
@@ -215,8 +215,8 @@ bool TemporaryThreatModifierEvent::Execute(uint64 /*e_time*/, uint32 /*p_time*/)
     return true;
 }
 
-Creature::Creature(bool isWorldObject): Unit(isWorldObject), MovableMapObject(), m_groupLootTimer(0), lootingGroupLowGUID(0), m_PlayerDamageReq(0), m_lootRecipientGroup(0),
-    m_corpseRemoveTime(0), m_respawnTime(0), m_respawnDelay(300), m_corpseDelay(60), m_wanderDistance(0.0f), m_boundaryCheckTime(2500),
+Creature::Creature(bool isWorldObject): Unit(isWorldObject), MovableMapObject(), m_groupLootTimer(0), lootingGroupLowGUID(0), _playerDamageReq(0),
+    m_lootRecipientGroup(0), _damagedByPlayer(false), m_corpseRemoveTime(0), m_respawnTime(0), m_respawnDelay(300), m_corpseDelay(60), m_wanderDistance(0.0f), m_boundaryCheckTime(2500),
```

Comment:
lets try moving _playerDamageReq(0) to after  m_lootRecipientGroup(0) and see how it plays from there.
![image](https://user-images.githubusercontent.com/16887899/174450801-1746bf40-f77d-4dd6-994c-10ee92794cc1.png)